src/components/DonwPanelButtonsCard/DownPanelCard.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import './DownPanelCard.css'
+import { BtnCard } from "../ButtonCards";
+
+import IconMarkCheked from '../Icons/IconMarkCheked.svg';
+import IconLike from '../Icons/IconLike.svg'
+import IconDislike from '../Icons/IconDislike.svg'
+import IconMark from '../Icons/IconMark.svg'
+import IconElipsis from '../Icons/IconElipsis.svg'
+
+type DownPanelCardProps = {
+    className?: string;
+};
+
+export const DownPanelCard = (props: DownPanelCardProps) => {
+
+    const [icon, setIcon] = useState(IconMark);
+
+    function checkedMark(){
+        if(icon === IconMark){
+            setIcon(IconMarkCheked);
+        }else{
+            setIcon(IconMark)
+        }
+    }
+
+    const [countLike, setCountLike] = useState(0);
+    const [countDislike, setCountDislike] = useState(0);
+
+    function actionButton(plusCount: any, count: number) {
+            plusCount(count + 1);
+    }
```

Comment:
Здесь лучше создать 2 разные функции для добавление и удаления лайков
```suggestion
    function onLike() {
            setCountLike(countLike + 1);
    }
```